src/NServiceBus.Transport.Msmq/Utils/ContextExtensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace NServiceBus.Transport.Msmq
+{
+    using Extensibility;
+
+    static class ContextExtensions
+    {
+        public static ReadOnlyContextBag GetOperationProperties(this ContextBag contextBag)
```

Comment:
```suggestion
        // Implements the 'ExtendableOptionsExtensions.GetOperationProperties' extension from Core to prevent this patch to force an update to Core 7.7 (as this has also been fixed on 7.5 and 7.6).
        public static ReadOnlyContextBag GetOperationProperties(this ContextBag contextBag)
```

Reply:
@cquirosj I think this might be a helpful comment to add. If you agree please commit before merging the PR.